### DISCUSSION TITLE
Diff preview cut file name when there is no need for it

### DISCUSSION THREAD

**Discussion by dev-masih** [community/community#138217] [Mobile]
Have a look at this image

![Screenshot_20240910_000426_GitHub.jpg](https://github.com/user-attachments/assets/2dbbeb04-50ee-4cfa-8320-ae792f4774d3)

The removed file at the end of the list got its name cut, the file name is grpc_startup.py and I'm sure it's not that long, even if it needs to get cut I expected it to get cut at the end of the file name not at the start of it.

I use the latest version of GitHub android app on Samsung Galaxy A15

**Comment by VadymMakohon** [comment]
1. Open the GitHub app on a Samsung Galaxy A15 device.
2. View a list of files in a commit, particularly looking at the removed files section.
3. Observe that a file with the name grpc_startup.py is truncated at the start instead of the end.
